src/api/app/views/webui/request/beta_show_tabs/_overview.html.haml

```diff
@@ -1 +1,6 @@
-Overview
+%h4.list-group.mb-4 Comments & Request History
+.comments-list{ data: { comment_counter: local_assigns[:comment_counter_id] } }
```

Comment:
I think this class name and the data attributes make no sense here now that the list contains both comments and history elements. However, I'm not going to require a change here because this is going to change in [this card](https://trello.com/c/GTxp0rnP/1940-handle-the-new-comments-and-the-replies-correctly-m) (confirmed with @danidoni).